ext/cl_khr_icd.asciidoc

```diff
@@ -28,6 +28,7 @@ implementation, the string *cl_khr_icd* will be present in the
 |====
 | *Date*     | *Version* | *Description*
 | 2020-04-21 | 1.0.0     | First assigned version.
+| XXXX-XX-XX | 2.0.0     | Loader managed dispatch.
```

Comment:
It might be informative to add a paragraph to the intro about the problems of 1.0.0 that 2.0.0 solves. A simple example exposing the problems (segfaults) would help understanding. Is it only related to unloading of the drivers/layers? It's not obvious to me who haven't been able to follow the ICD2 discussion closely, thus it likely won't be obvious to others new to the extension text.

Reply:
Will do, thanks for the suggestion. I will add a rationale section.

Reply:
Added in the issues section.